client/src/js/components/bhStockFilterExhausted.js

```diff
@@ -0,0 +1,34 @@
+angular.module('bhima.components')
+  .component('bhStockFilterExhausted', {
+    bindings : {
+      value : '<',
+      label : '@',
+      name : '@',
+      excludeLabel : '@',
+      includeLabel : '@',
+      includeOnlyLabel : '@',
+      helpText : '@?',
```

Comment:
Is it possible to remove these labels : label, excludeLabel, includeLabel, includeOnlyLabel and helpText. And just insert these labels directly in the template file since these labels will not be changed often, I think.

Reply:
I agree; I had already thought of that.  But I put them in there in case we wanted to use different language for Stock > Lots vs Stock > Articles in Inventory.  If you think that is unnecessary, I'm happy to do what you suggest.  Note that using the same language in both right now was only for convenience.

Reply:
Ok, I did this.  We can always add it back in if we need to use different language later.